roles/openshift_router/tasks/main.yml

```diff
@@ -0,0 +1,11 @@
+---
+
+# TODO: Centralized openshift_config openshift_facts??
+# TODO: anchor this on a master
+- name: Query for OpenShift router
+  command: openshift ex router --credentials={{ openshift_config }}
```

Comment:
We need to discuss the --images flag in regards to having the cluster use the self-hosted docker registry for pods. We will need some administrator driven method to populate that registry from registry.access.redhat.com as needed.  That way the admin has control over which versions of which images are available for the cluster.